content/blog/working-with-benk-a-storage-provisioning-and-io-performance-benchmark-suite-for-kubernetes.md

```diff
@@ -0,0 +1,232 @@
+
+Let’s change block size for our previous workload example. We’re going to summarize if increasing the block size will increase the bandwidth for the workload.
+
+Change the “workloadBlockSize” to “64k” with the `vi` ceremony.
+
+```text
+vi kustomize/overlays/mytest-*/config.env
+```
+
+Re-run the sequencer.
+
+```text
+./sequencer.sh mytest-
+```
+
+We’ll have to use a different template as Jinja2 now has to deal with managing two log files at a time. The syntax for the `outputter.py` script is also slightly different.
+
+```text
+./src/benk/outputter.py -a logs/run-mytest-<unique timestamp>.log -b logs/run-mytest-<unique timestamp>.log -t jinja2/example-default-ab.md.j2
+```
+
+This report template will summarize the findings in a markdown table, suitable to include in a GitHub pull request or similar to illustrate a certain discovery.
+
+```text
+| Threads | A (MB/s) | B (MB/s) | Diff |
+| ------- | -------- | -------- | ---- |
+| 1       | 13       | 55       | 4.2x |
+| 2       | 26       | 106      | 4.1x |
+| 4       | 63       | 181      | 2.9x |
+| 8       | 94       | 450      | 4.8x |
+| 16      | 128      | 661      | 5.2x |
+| 32      | 135      | 748      | 5.5x |
+| 64      | 137      | 840      | 6.1x |
+| 128     | 147      | 833      | 5.7x |
+```
+
+Clearly, the performance increases nearly 5x across the board and we also have a discrepancy in the dataset. Which one?
+
+A good exercise for the reader could be to factor in latency in the report to understand the impact, which there will be as performance plateaus when you add more workload usually result in higher latency. How high?
+
+# More sequencers and examples!
+
+There’s also a `sequencer-cluster.sh` script that allows users to orchestrate load generation across multiple clusters attached to one or many storage systems to isolate problems with high concurrency. The possibilities are quite endless.
+
+Learn more about multi-cluster scaling in [the GitHub repository](https://github.com/hpe-storage/benk#multi-cluster-testing).
+
+You'll also find more practical examples in [the GitHub repository](https://github.com/hpe-storage/benk/tree/main/examples) stemming from real world performance testing conducted by the HPE Hybrid Cloud solutions team.
+
+# Summary
+
+This hopefully gets you started on ideas you’d like to build and use cases you want to explore. The possibilities are endless. Bear in mind that Benk is by all means provided as-is and a very early implementation HPE chose to open source to better collaborate with customers and partners to isolate performance bottlenecks with full transparency. Not all advertised features in `config.env` has been implemented yet as an example and the CLI is not completed yet.
```

Comment:
Modify first sentence to:
"Hopefully, this blog post gets you started..."

Place commas around "by all means" in the second sentence. I would also divide out that second sentence into:

"Bear in mind that Benk is, by all means, provided as-is. In addition, to be fully transparent, this is a very early implementation that HPE chose to open source in order to better collaborate with customers and partners to isolate performance bottlenecks." 

Then, modify the last sentence to read:
 "For example, not all advertised features in config.env have been implemented yet and the CLI is not yet completed."